.drone.star

```diff
@@ -1627,6 +1627,8 @@ def ocisServer(storage, accounts_hash_difficulty = 4, volumes = [], depends_on =
             "OCIS_LOG_LEVEL": "error",
             "SETTINGS_DATA_PATH": "/srv/app/tmp/ocis/settings",
             "OCIS_INSECURE": "true",
+            "ACCOUNTS_DEMO_USERS_AND_GROUPS": True,  # deprecated, remove after switching to LibreIDM
+            "IDM_CREATE_DEMO_USERS": True,
```

Comment:
I don't think we need those here, because the tests should create all needed users themself

Reply:
they need `moss` as an admin user.

Reply:
I tried it without the demo users, didn't work

Reply:
moss is part of the demo users. I don't remember the reason why we didn't use `admin:admin`

Reply:
> they need moss as an admin user.

where is that?

the only place where we are using the demo users I can find is here https://github.com/owncloud/ocis/blob/03cadf304c650927b04f244ba7be84fb47eccad2/accounts/ui/tests/acceptance/features/accounts.feature#L4
we should create users in that feature